cmd/gomobile/env.go

```diff
@@ -39,7 +39,7 @@ func allArchs(targetOS string) []string {
 func allTargets(targetOS string) []string {
 	switch targetOS {
 	case "ios":
-		return []string{"simulator", "ios", "catalyst"}
+		return []string{"simulator", "ios", "catalyst", "macosx"}
```

Comment:
It’s entirely nitpicky, but `”macos”` would be better than `”macosx”`, since Big Sur isn’t “X” anymore :-)

Reply:
Yes, you’re right, but `macosx` is the literal name of the SDK:

```
~/development/ydnar/gomobile support-catalyst*
❯ xcrun --sdk macosx --show-sdk-path
/Applications/Xcode-beta.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.0.sdk

~/development/ydnar/gomobile support-catalyst*
❯ xcrun --sdk macos --show-sdk-path
2021-07-08 10:53:02.844 xcodebuild[71964:4908259] [MT] DVTPlugInManager: Required plug-in compatibility UUID 42E1F17B-27B3-4DE8-92A8-DC76BA4F5921 for DVTCoreGlyphs.framework (com.apple.dt.DVTCoreGlyphs) not present
xcodebuild: error: SDK "macos" cannot be located.
2021-07-08 10:53:03.728 xcodebuild[71965:4908283] [MT] DVTPlugInManager: Required plug-in compatibility UUID 42E1F17B-27B3-4DE8-92A8-DC76BA4F5921 for DVTCoreGlyphs.framework (com.apple.dt.DVTCoreGlyphs) not present
xcodebuild: error: SDK "macos" cannot be located.
xcrun: error: unable to lookup item 'Path' in SDK 'macos'
```

Reply:
Admittedly true, but Go does have a history of using more abstracted platform names. It makes sense to do so because it makes support feel less stale, e.g. `"windows"` instead of `"win32"`.

Reply:
I see your point. Since this isn’t passed directly to `xcrun`, it could probably be `macos` instead of `macosx`.